Package.swift

```diff
@@ -30,30 +33,106 @@ let package = Package(
         .package(
             name: "Quick",
             url: "https://github.com/Quick/Quick.git", from: "3.0.0"),
-    ],
+    ] + environment.dependencies,
     targets: [
-        .binaryTarget(
-            name: "MaasCore",
-            // Used for production
-//            url: "https://github.com/trafi/maas-components/releases/download/0.1.0-dev05/MaasCore.xcframework.zip",
-//            checksum: "d0e9dfa86527369490cb3293e6370e3d8844d14cd35a19bde44433fc337921ed"),
-            // Used for development
-            path: "common/core/build/bin/xcframework/MaasCore.xcframework"),
-
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages this package depends on.
         .target(
-            name: "MaaS",
-            dependencies: ["MaasCore", "Swappable", "MaasTheme"],
-            path: "ios/Sources/MaaS"),
-
+            name: "Maas",
+            dependencies: ["Swappable"],
+            path: "ios/Sources/Maas"),
+        .target(
+            name: "Examples",
+            dependencies: [
+                "MaasNearbyTransit",
+                "MaasRouteSearch"
+            ],
+            path: "ios/Sources/Examples"
+        ),
         .testTarget(
             name: "MaasTests",
-            dependencies: ["MaaS", "SnapshotTesting", "Quick"],
+            dependencies: [
+                "Maas",
+                "MaasNearbyTransit",
+                "MaasRouteSearch",
+                "SnapshotTesting",
+                "Quick"
+            ],
             path: "ios/Tests/MaasTests",
-            exclude: ["__Snapshots__"]),
-
-        .target(
-            name: "MaasTheme",
-            dependencies: ["MaasCore"],
-            path: "ios/Sources/MaasTheme"),
-    ]
+            exclude: ["__Snapshots__"]), 
+    ] + environment.targets 
 )
+
+// MARK: - Setup Extension
+
+private extension Environment {
+
+    var products: [Product] {
+        switch self {
+        case .development:
+            return [
+                // --
+            ]
+        case .production:
+            return [
+                .library(
+                    name: "MaasCore",
+                    targets: ["MaasCore"]),
+               .library(
+                   name: "MaasNearbyTransit",
+                   targets: ["MaasNearbyTransit"]),
+               .library(
+                   name: "MaasRouteSearch",
+                   targets: ["MaasRouteSearch"]),
+           ]
+        }
+    }
+
+    var targets: [Target] {
+        switch self {
+        case .development:
+            return [
+                // --
+            ]
+        case .production:
+            return [
+                .binaryTarget(
+                    name: "MaasCore",
+                    path: "ios/MaasCore/Sources/MaasCore/Core.xcframework"),
```

Comment:
We're not committing the xcframework to the repository, so eventually this will need to be a URL for production and a local path as here to development.

Reply:
👍